gradle.properties

```diff
@@ -1,20 +1,17 @@
+projectVersion=6.0.0-SNAPSHOT
+
+asciidoctorJvmVersion=4.0.3
 grailsVersion=7.0.0-SNAPSHOT
 grailsGradlePluginVersion=7.0.0-SNAPSHOT
-groovyVersion=4.0.22
-liquibaseHibernate5Version=4.27.0
-projectVersion=6.0.0-SNAPSHOT
-hibernate5Version=5.6.15.Final
-servletApiVersion=6.0.0
+groovyVersion=4.0.23
+# Tests are built for this version
 h2Version=1.4.200
```

Comment:
h2 is part of the bom.

Reply:
I think the `h2` version is held back on purpose here, considering the comment above.

Reply:
It is, but the tests are based on this version so it is overridden.

Reply:
how are tests based on a version of a database?

Reply:
There are compatibility issues with `h2:2.x`. #385

Reply:
well, hopefully we will get to updating them at some point..  My h2 issues always have been pretty straightforward upgraades with the exception of cases where I had to do a persisted database update.

Reply:
I have updated the tests in for `h2:2.x` but I'll wait for this PR to merge before creating a PR because there are some changes here I need.